test/init.js

```diff
@@ -22,19 +23,79 @@ var config = {
 console.log('env config ', config);
 
 global.config = config;
+global.IMPORTED_TEST = false;
 
 global.getDataSource = global.getSchema = function(customConfig) {
   var db = new DataSource(require('../'), customConfig || config);
   db.log = function(a) {
     console.log(a);
   };
+
+  var originalConnector = _.clone(db.connector);
+  var overrideConnector = {};
+
+  overrideConnector.automigrate = function(models, cb) {
```

Comment:
Ok, I see there is this function. But again my question, why not remove `db.once('connected', cb)` from all the test files since they will be checked in here?